### DISCUSSION TITLE
I find Meshlab really useful for processing point clouds!

### DISCUSSION THREAD

**Discussion by UnknownTarget** [cnr-isti-vclab/meshlab#884] [Show and tell]
Unfortunately I can't post pictures because they're dense point clouds of people's faces, generated through photogrammetry. Agisoft Metashape is having trouble meshing the cloud so I'm using Meshlab to prep it for volumetric surface generation in Grasshopper.

Application is bespoke masks based on scans. https://elevons.design/masksquared/

**Comment by alemuntoni** [comment]
Thanks for sharing! :)
